script/Ciel/Day 10/10_01_CIEL10_6.txt

```diff
@@ -481,87 +530,95 @@ The vampires in that underground garage for instance. Those guys were above rank
 [sha:52f6d4550d8c60becb5e6b2c9a1c745996345e74]{
 -- Page 452, Offset 28332.
 -- 「テレビでもネットでもいい。ここ数日で身元不明の遺体を発見した、というニュースはありましたか？
-"On TV or on the internet, it doesn't matter. Was there any report those last few days of unidentified corpses having been found?
+"It doesn't matter whether it was on TV or the internet.
+ Have you seen any reports in the past couple of days about unidentified corpses turning up?
 }
 [sha:8d96c8527ccd2979dd5db553096c9afa18e5346e]{
 -- Page 452, Offset 28333.
 -- ロア……親である死徒が消滅したのなら、その時点で隠蔽されていた事件が明るみになる。
-If Roa... if the sire, had vanished, all the cases that had up to now been hideen would have come to light.
+If Roa... if the sire really vanished, then all of the disappearances they've been covering up should have come to light.
 }
 [sha:2618b8ddcaae7ef48ee5ac3f0d8fd2f29788606b]{
 -- Page 452, Offset 28334.
 -- 死徒討伐において、<街|・><が|・><ま|・><だ|・><平|・><穏|・><で|・><あ|・><る|・><事|・>―――それ自体が何も解決していない証拠なんです……！」
-When exterminating Dead Apostles, %{i}for the city to still be at peace%{/i}―――That alone is proof that nothing's been solved...!"
+If you're exterminating Dead Apostles,
+ and the city %{i}continues to be peaceful%{/i},
+ that's actually proof that you've yet to accomplish anything...!"
 }
 [sha:129b1899806fbd52ea61f52c6260c3c97eb6e31b]{
 -- Page 453, Offset 28335.
 -- 「―――じゃ、じゃあ。
-"T-Then―――
+"T-Then...
 }
 [sha:f05056f7aa8b2a27c29d35eedd1102b798fc29d7]{
 -- Page 453, Offset 28336.
 -- そこにあるロアの遺体は偽物で、ロアのヤツは、」
-The dead corpse here isn't actually Roa's, and Roa is..."
+You mean that corpse over there is a fake, and Roa's actually―――"
 }
 [sha:1caf0c96251a488a4086a501245d2aa4fc8486d2]{
 -- Page 454, Offset 28337.
 -- 「そう。死徒ロアはまだこの街にいる。
-"Yes. Dead Apostle Roa is still in the city.
+"Yes. The Dead Apostle, Roa, is still in this city.
 }
 [sha:05453c74abba5c913e537ed48f47b4fad57d10da]{
 -- Page 454, Offset 28338.
 -- いえ―――はじめからずっと同じ<器|からだ>にいたんじゃないかしら。ねえ、そう思わない、遠野志貴クン？」
-No... He's always been here, in the vessel he had since the very beginning, I imagine. Hmm, am I right, Tohno Shiki-kun?"
+In fact... I'd imagine he's actually been in the same body this entire time.
+ What do you think, am I right, Tohno Shiki-kun?"
```

Comment:
I guess what are we trying to suggest with that?

Reply:
She says his full name for once because she's emphasizing his lineage, was the thought.

Reply:
Hmm, it doesn't really sound like she's emphasizing it in the VA that much, and since we use _Shiki_ to refer to シキ I'd be worried about causing undue confusion here.